src/main/java/org/edumips64/ui/swing/SplashScreen.java

```diff
@@ -0,0 +1,71 @@
+/* SplashScreen.java
+ *
+ * SplashScreen for EduMIPS64
+ * (c) 2006 Andrea Spadaccini, Antonella Scandura, Vanni Rizzo
```

Comment:
You should put your name (or username) here and the year 2020.

Reply:
Done, this one and the other two pending have been corrected.